portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -49,26 +50,21 @@ private String getComment(HttpServletRequest request) {
 
 @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String rn = request.getParameter("quantity");
-    int requestedNum = 5;
-
-    if (rn!=null) {
-        requestedNum = Integer.parseInt(rn);
-    }
+    String requestedNumString = request.getParameter("commentRequestedNum");
+    int requestedNumInt = Integer.parseInt(requestedNumString);
```

Comment:
Inline

int requestedNumInt = Integer.parseInt(request.getParameter("commentRequestedNum"));

Reply:
Inline

Reply:
This change didn't seem to propogate?

Reply:
Propogated